trunk/SUAVE/Components/Energy/Converters/Internal_Combustion_Engine.py

```diff
@@ -0,0 +1,102 @@
+# Internal_Combustion_Engine.py
+#
+# Created:  Aug, 2016: D. Bianchi
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+
+# suave imports
+import SUAVE
+from SUAVE.Core import Data, Units
+
+# package imports
+import numpy as np
+from SUAVE.Components.Energy.Energy_Component import Energy_Component
+
+# ----------------------------------------------------------------------
+#  Internal Combustion Engine Class
+# ----------------------------------------------------------------------
+
+class Internal_Combustion_Engine(Energy_Component):
+
+    def __defaults__(self):
+
+        self.sea_level_power    = 0.0
+        self.flat_rate_altitude = 0.0
+        self.speed              = 0.0
+        self.BSFC               = 0.36 # lb/hr/hp :: Ref: Table 5.1, Modern diesel engines, Saeed Farokhi, Aircraft Propulsion (2014)
```

Comment:
Write this out, especially since it's written out in the output.

Reply:
BSFC spelled out to "power_specific_fuel_consumption"